docs/develop/dapps/apis/sdk.mdx

```diff
@@ -19,6 +19,7 @@ There are different ways to connect to blockchain:
 |[tonweb](https://github.com/toncenter/tonweb)|via RPC ([Orbs](https://www.orbs.com/ton-access/) / [Toncenter](https://toncenter.com/api/v2/) / etc)|Old-style TON JS SDK, with minimal external dependencies, extensively tested in production.|
 |[tonkite/adnl](https://github.com/tonkite/adnl)|[ADNL](/develop/network/adnl-tcp) natively / via WebSocket| ADNL TypeScript implementation. |
 |[tonutils](https://github.com/thekiba/tonutils)|Native [ADNL](/develop/network/adnl-tcp)| TypeScript-based interface for building and interacting with applications in TON Ecosystem. Due to native ADNL dependency, cannot be used for blockchain interaction in browser.|
+|[foton](https://foton.sh/)|via RPC ([Orbs](https://www.orbs.com/ton-access/) / [Toncenter](https://toncenter.com/api/v2/) / etc)|TypeScript toolkit for interacting with TON wallets and blockchain as a whole. The library wraps existing solutions (Blueprint and TON Connect) into one comfortable API.|
```

Comment:
```suggestion
|[Foton](https://foton.sh/)|via RPC ([ORBS](https://www.orbs.com/ton-access/) / [Toncenter](https://toncenter.com/api/v2/) / etc)|TypeScript toolkit for interacting with TON wallets and blockchain as a whole. The library wraps existing solutions (Blueprint and TON Connect) into one comfortable API.|
```

Reply:
In other cases, don't need to do Orbs => ORBS?

Reply:
This is an interesting question

Brand definitions are all capitalized
https://www.orbs.com/brand-assets/


But you will find that the name on Twitter only has the first character capitalized
https://x.com/orbs_network

Well, I don't know which one is right.